packages/collector/src/logger.js

```diff
@@ -102,6 +102,9 @@ exports.getLogger = function getLogger(loggerName, reInitFn) {
   }
 
   if (reInitFn) {
+    if (registry[loggerName]) {
+      throw new Error(`Duplicate logger name: ${loggerName}.`);
+    }
```

Comment:
While working on the test for the sqs change, I noticed by chance that we have a few wrong logger names in the `getLogger('tracing/mongoose', newLogger => { ... });` calls, due to copy&paste errors. Using the same module logger name twice breaks the `instana.setLogger` call - one of the two modules that used the same logger name will not receive the new logger instance.

I know our mantra is "do not throw errors" but I am confident that this will only ever be thrown at development time when accidentally using a module logger name twice. No application under test will be able start so all tests will be red when this error condition exists.

WDYT?

Reply:
Basically all integration tests will fail once a module does that. We always load/require all modules at the start of the application, so no matter which integration test is executed, the error will be thrown in the application under test. Failing ~80% of our test cases should be a good enough signal to fix the problem I guess.

Reply:
Cool 👍